distribution/docs/src/main/resources/_contents/running-contents.adoc

```diff
@@ -91,19 +91,24 @@ to:
 RUN_AS_USER=<${ddf-branding-lowercase}-user>
 ----
 +
-. Set the memory in the wrapper config to match with ${branding} default memory setting.
-.. Add the setting for PermGen space under the JVM Parameters section.
-.. Update heap space to 2048.
```

Comment:
Permgen is no longer a thing in JDK 8  (they have a new memory model that
does not use it).    Since DDF required 8 I didn't think it was necessary.

Heap space is still being set to 2048

On Thu, Aug 11, 2016 at 3:50 PM, Michael Menousek <notifications@github.com>
wrote:

> In distribution/docs/src/main/resources/_contents/running-contents.adoc
> <https://github.com/codice/ddf/pull/1133#discussion_r74490744>:
>
> > @@ -91,19 +91,24 @@ to:
> >  RUN_AS_USER=<${ddf-branding-lowercase}-user>
> >  ----
> >  +
> > -. Set the memory in the wrapper config to match with ${branding} default memory setting.
> > -.. Add the setting for PermGen space under the JVM Parameters section.
> > -.. Update heap space to 2048.
>
> Any reason why these were deleted?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/codice/ddf/pull/1133/files/7aab8a7450c1e05708fa15f707303f51b48b1c62#r74490744>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADb7lXGpwRUMCk5uidfqvgeBvIWDYda0ks5qe30SgaJpZM4JicAS>
> .
>